app/src/main/res/values/strings.xml

```diff
@@ -288,7 +288,7 @@
   <!-- ResetPinDialogFragment -->
   <string name="reset_pin_enter">Enter a New Pin</string>
   <!-- MyDownloadsActivity -->
-  <string name="my_downloads">My Downloads</string>
+  <string name="my_downloads_activity_title">My Downloads</string>
```

Comment:
Can you please also rename the corresponding strings in our translated strings.xml files, too?